src/HeadersParser.h

```diff
@@ -243,12 +245,6 @@ namespace snowcrash {
                         headerSM.sourceMap = node->sourceMap;
                         out.sourceMap.collection.push_back(headerSM);
                     }
-                } else {
-                    // WARN: unable to parse header
-                    mdp::CharactersRangeSet sourceMap = mdp::BytesRangeSetToCharactersRangeSet(node->sourceMap, pd.sourceCharacterIndex);
```

Comment:
Because you would get two error messages for one issue and after running the test suite it seemed unnecessary but apparently this has not been tested at all.